src/main/java/org/opensearch/neuralsearch/search/query/HybridCollectorManager.java

```diff
@@ -118,6 +121,27 @@ public Collector newCollector() {
      */
     @Override
     public ReduceableSearchResult reduce(Collection<Collector> collectors) {
+        final List<HybridTopScoreDocCollector> hybridTopScoreDocCollectors = getHybridScoreDocCollectors(collectors);
+        if (hybridTopScoreDocCollectors.isEmpty()) {
```

Comment:
Can we move this validation into getHybridScoreDocCollectors method itself?

Reply:
I don't think it's a proper place for that validation. getter is responsible only for giving the collection of collectors, usage of that collection including validation is client responsibility